platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -32,20 +35,29 @@
 public class CrlChecker {
 
     public static final String CRL_PROPERTY_KEY = Merlin.OLD_PREFIX + Merlin.X509_CRL_FILE;
+    
+    public static final Path ENCRYPTION_PROPERTIES_PATH = Paths.get("etc", "ws-security", "server", "encryption.properties");
```

Comment:
You might need to also read signature.properties and also the properties files under issuer. There is no guarantee that the CRL will be the same between them. It _probably_ will be, but you still don't know.